bert_reranker/main.py

```diff
@@ -56,10 +59,16 @@ def main():
         hyper_params = load(stream, Loader=yaml.FullLoader)
 
     check_and_log_hp(
-        ['train_file', 'dev_files', 'test_file', 'cache_folder', 'batch_size', 'tokenizer_name',
-         'model', 'max_question_len', 'max_paragraph_len', 'patience', 'gradient_clipping',
-         'loss_type', 'optimizer',  'precision', 'accumulate_grad_batches'],
+        ['train_file', 'dev_files', 'test_file', 'cache_folder', 'batch_size', 'tokenizer_name', 'model',
+         'max_question_len', 'max_paragraph_len', 'patience', 'gradient_clipping',
+         'loss_type', 'optimizer',  'precision', 'accumulate_grad_batches', 'seed'],
         hyper_params)
+    
+    # fix the seed
```

Comment:
can you run this if the seed is not None?
so that we can still run without setting a seed (and using cudnn optimizations)

Reply:
ah ok I see what you say